package.json

```diff
@@ -158,9 +165,13 @@
     "eslint-plugin-react": "^7.31.10",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-unused-imports": "^2.0.0",
+    "mochawesome": "^7.1.3",
     "prettier": "^2.7.1",
     "react-scripts": "^4.0.3",
     "source-map-explorer": "^2.5.3",
     "typescript": "^4.8.4"
+  },
+  "resolutions": {
+    "ethers": "^5.0.0"
```

Comment:
seems dangerous

Reply:
@namgold : the libraries for test module

Reply:
This might causing break since this impacts the the codebase. We need to test this carefully.